scripts/standardising.py

```diff
@@ -53,18 +52,9 @@ def standardising(file: str, preset: str) -> str:
 
 
 def main() -> None:
-    concurrency: int = 1
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--preset", dest="preset", required=False, default="lzw")
-    parser.add_argument("--source", dest="source", nargs="+", required=True)
-    arguments = parser.parse_args()
 
-    source = format_source(arguments.source)
-
-    if is_argo():
-        concurrency = 4
-
-    start_standardising(source, arguments.preset, concurrency)
+    source = parse_source()
+    start_standardising(source, "lzw", 1)
```

Comment:
shouldn't this also be able to run with webp and mutlithreaded?

Reply:
I think we agreed to revisit that when there was a need to run standardise alone. Otherwise mypy complains about code duplication.

Reply:
mypy or pylint?

Reply:
is it time to disable that lint rule? seems very onerous

Reply:
another approach would be to allow x lines of duplicated code

Reply:
not allowing 4 lines of duplicated code seems like a overly strict rule?

Reply:
I've reverted the `standardising` script to take arguments, `pylint` seems happy so :shrug: